lib/main.js

```diff
@@ -503,6 +506,7 @@ function _saveWindowStates () {
       url: winInfo.url || win.webContents.getURL(),
       argv: winInfo.argv,
       userdata: winInfo.userdata,
+      fullscreen: win.isFullScreen(),
```

Comment:
I think we should only save the fullscreen state for mainWin (yes i know the fullscreen in the above code only affect mainWin, i think explicit declare this condition will make things clear). may be better:

```
fullscreen: win === _mainwin ? win.isFullScreen() : false,
```

Reply:
Hey Johnny -- 

You got it. I made that change for you. I left in the first if statement at the top, because if we assign fullscreen a false value, it disables the fullscreen button entirely. 

It should be good to go now!